irc_channel.c

```diff
@@ -625,6 +625,24 @@ static gboolean control_channel_invite( irc_channel_t *ic, irc_user_t *iu )
 	return TRUE;
 }
 
+static void control_channel_kick( irc_channel_t *ic, irc_user_t *iu )
+{
+	struct irc_control_channel *icc = ic->data;
+	bee_user_t *bu = iu->bu;
+	
+	if( bu == NULL )
+		return;
+	
+	if( icc->type != IRC_CC_TYPE_GROUP )
+	{
+		irc_send_num( ic->irc, 482, "%s :Kicks are only possible to fill_by=group channels", ic->name );
```

Comment:
What would you expect /invite to do in other cases? For fill_by=group, it can add a contact to a group so that the change persists through sessions. For anything else, the /invite can't really be mapped to IM space (can't add a FB account to a GTalk contact list, etc.).